apollo-router/tests/integration_tests.rs

```diff
@@ -111,7 +111,7 @@ async fn api_schema_hides_field() {
     assert!(actual.errors[0]
         .message
         .as_str()
-        .contains("Cannot query field \"inStock\" on type \"Product\"."));
+        .contains("cannot query field `inStock` on type `Product`"));
```

Comment:
I seeeeeee. this error now comes from apollo-rs validation instead of the router's code. We could totally convert this particular error to the existing router error type, but there will still be other validation errors that differ between apollo-rs and graphql-js.

Reply:
We also should have extension code, could you check that it's correctly set please ?